portfolio/src/main/webapp/script.js

```diff
@@ -15,8 +15,11 @@
 /**
  * Adds a random greeting to the page.
  */
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(experienceChart);
 
-var count = 0;
+let count = 0;
+var map;
```

Comment:
Can you use let instead? https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#:~:text=%60const%60%20is%20a%20signal%20that,always%20the%20entire%20containing%20function.